release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3111,6 +3111,19 @@ You can view the container images in this release by running the following comma
 ----
 $ oc adm release info 4.12.0 --pullspecs
 ----
+
+[id="ocp-4-12-0-ironic-agent-image-ztp"]
+==== Ironic agent mirroring in disconnected GitOps ZTP installations
+
+For disconnected installations using GitOps ZTP, if you are deploying {product-title} version 4.11 or earlier to a spoke cluster with converged flow enabled, you must mirror the default Ironic agent image to the local image repository. The default Ironic agent images are the following:
+
+* AMD64 Ironic agent image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d3f1d4d3cd5fbcf1b9249dd71d01be4b901d337fdc5f8f66569eb71df4d9d446
+
+* Aarch64 Ironic agent image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cb0edf19fffc17f542a7efae76939b1e9757dc75782d4727fb0aa77ed5809b43
```

Comment:
I don't know about AArch64, just basing that on the other PR of yours I saw today :)
```suggestion
* AArch64 Ironic agent image: `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cb0edf19fffc17f542a7efae76939b1e9757dc75782d4727fb0aa77ed5809b43`
```